distribution/lib/Standard/Table/0.0.0-dev/src/Io/File_Format.enso

```diff
@@ -63,6 +74,11 @@ type Plain_Text
     read file on_problems =
         file.read_text this.encoding on_problems
 
+    ## Implements the `Table.write` for this `File_Format`.
+    write_table : File -> Table -> Existing_File_Behavior -> Column_Mapping -> Problem_Behavior -> Nothing
+    write_table _ _ _ _ _ =
+        Error.throw (Illegal_Argument_Error "Saving a Table as Plain_Text is not directly supported. You may convert the Table to a Text using `Text.from` and then use `Text.write` to write it.")
```

Comment:
Do we have a Text.from with File_Format.Delimited yet? If not can we add a task as I think I forgot that path.

Reply:
Yeah, I don't think we have. I can add it.